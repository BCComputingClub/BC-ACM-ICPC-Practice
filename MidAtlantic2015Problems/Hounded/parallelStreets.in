10 5
XXXXXXXXXX
ET       E
X XXXXXXXX
X  K     X
XXXXXXXXXX
10 5
XXXXXXXXXX
E T      E
X XXXXXXXX
X  K     X
XXXXXXXXXX
10 5
XXXXXXXXXX
E  T     E
X XXXXXXXX
X  K     X
XXXXXXXXXX
10 5
XXXXXXXXXX
E   T    E
X XXXXXXXX
X  K     X
XXXXXXXXXX
10 5
XXXXXXXXXX
E    T   E
X XXXXXXXX
X  K     X
XXXXXXXXXX
20 5
XXXXXXXXXXXXXXXXXXXX
E    T             E
X XXXXXXXXXXXXXXXXXX
X  K               X
XXXXXXXXXXXXXXXXXXXX
21 5
XXXXXXXXXXXXXXXXXXXXX
E    T              E
X XXXXXXXXXXXXXXXXXXX
X  K                X
XXXXXXXXXXXXXXXXXXXXX
22 5
XXXXXXXXXXXXXXXXXXXXXX
E    T               E
X XXXXXXXXXXXXXXXXXXXX
X  K                 X
XXXXXXXXXXXXXXXXXXXXXX
0 0
