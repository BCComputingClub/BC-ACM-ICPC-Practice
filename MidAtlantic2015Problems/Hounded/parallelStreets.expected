KEEP IT
KEEP IT
KEEP IT
KEEP IT
KEEP IT
KEEP IT
KEEP IT
DROP IT
